src/native/corehost/apphost/standalone/CMakeLists.txt

```diff
@@ -41,11 +41,14 @@ include(../../exe.cmake)
 
 add_definitions(-DFEATURE_APPHOST=1)
 
-# Disable manifest generation into the file .exe on Windows
 if(CLR_CMAKE_TARGET_WIN32)
-    set_property(TARGET ${PROJECT_NAME} PROPERTY
-            LINK_FLAGS "/MANIFEST:NO"
-        )
+    # Disable manifest generation into the file .exe on Windows
+    add_linker_flag("/MANIFEST:NO")
+
+    # Enable CET-compatibility
+    if (CLR_CMAKE_HOST_ARCH_AMD64)
+        add_linker_flag("/CETCOMPAT")
+    endif()
```

Comment:
I'm not sure if this will add the flags to the target as the variables set by `add_linker_flag` only take effect on targets declared after the call (which is why the existing one is using such a weird syntax, it's adding it to the target after the fact)

Reply:
`target_link_options` is the more modern option, but if add_linker_flag works, then it's fine to use that.